.gitignore

```diff
@@ -1,4 +1,4 @@
-/target
+*target/
```

Comment:
We shouldn't have `target` dirs in any subdirectories.

Reply:
And I don't get it. 
The discussion is about `db/...` but I don't see any of it here... ?

Reply:
Removed the `db/`, we create `fedimint-client/target/`

Reply:
> Removed the `db/`, we create `fedimint-client/target/`

How... why? `fedimint-client` seems like a part of the whole workspace ...

Reply:
I did have `fedimint-client/target/`. I deleted it. Did `just build` and I no longer see it. Maybe it was already fixed?

Reply:
`just test` seems to produce it for me

Reply:
```
fedimint-client/target/doc/mermaid.min.js
```

Oh... I see.

Reply:
https://github.com/mersinvald/aquamarine/blob/636675ba342a37a2a0aa87eb9adeea5c266f505c/src/attrs.rs#L138 :dizzy_face:

Reply:
https://github.com/mersinvald/aquamarine/issues/38

Reply:
Let's go with this for now till upstream is fixed